### DISCUSSION TITLE
Welcome to the Elefant CMS discussions!

### DISCUSSION THREAD

**Discussion by jbroadway** [jbroadway/elefant#305] [General]
## 👋 Welcome!

It's been some time since the old Elefant CMS community forum has been down (upgrading Vanilla Forums didn't go so well), so we're super excited to be part of the GitHub Discussions beta!

We'll be using Discussions as a place for community members to connect with each other. We hope that you:

* Ask questions if you need help
* Share ideas for new features
* Engage with other community members
* Welcome others and be open-minded. Remember that this is a community we build together 💪

To get started, comment below with an introduction of yourself and tell us about what you do with Elefant.

**Comment by twheel** [comment]
Great to see the forums restarted!  I have a few elefant sites.  I still think it’s the most developer-friendly platform I have used.

**Reply by jbroadway** [reply]
I'm a little biased to make that claim myself, but I'm glad to hear that! 😄 

I'm using Elefant as the backend for [Flipside XR](https://www.flipsidexr.com/) as well as a new web app I'm building called [HeyAlfa](https://www.heyalfa.com/) (along with a few other sites), so I'm very dependent on keeping it up even though it can be hard to balance time on the framework vs features that directly affect one of those two apps. Overall, I'm pretty happy with recent improvements though.

**Reply by kreativmind** [reply]
You nean you're still alive @jbroadway ?

**Reply by jbroadway** [reply]
Spread way too thin between projects but still breathing, yes 😄

**Reply by kreativmind** [reply]
I understand. I have seen the Flipside project which is really cool. Would love to try it sometime. Haven't spoken to you in ages. Ever speak with Jarred from the old forum?

**Reply by jbroadway** [reply]
Not in quite some time (@shortjared, you still around?). I was hoping a few people from the old forum would find their way here but also nervous people wouldn't. Glad to see you guys found it right away!

**Reply by kreativmind** [reply]
Of course bro. Haven't forgot about you. :-) I can honestly say I haven't learned much more than before but I am still trying. Been too busy building webservers and a few sites. Been focused on ooloo.me and also Upostia.com (which I'd like to build from scratch vs WP site).

**Reply by jbroadway** [reply]
Nice, I love the idea of a site dedicated to building a community around writing!

**Reply by kreativmind** [reply]
I love to write is why I am doing it. I'd love to make it more of a platform and even have a mobile app for it. I have a lot to learn first though.
